pkg/sql/crdb_internal.go

```diff
@@ -3229,7 +3228,14 @@ CREATE TABLE crdb_internal.zones (
 
 			// Inherit full information about this zone.
 			fullZone := configProto
-			if err := completeZoneConfig(&fullZone, config.SystemTenantObjectID(tree.MustBeDInt(r[0])), getKey); err != nil {
+			// TODO(zcfg-pod): We currently lack introspection for zone configurations
+			// for secondary tenants.
```

Comment:
Why? We have the codec available to us: `p.ExecCfg().Codec`.

Reply:
Shouldn't it "just work" if we got rid of the `if !p.ExecCfg().Codec.ForSystemTenant()` check above?